packages/core/docs/core/getting-started.md

```diff
@@ -30,11 +30,17 @@ During the installation process you will be asked questions about:
 
 Based on provided answers Vue Storefront CLI will generate a project already integrated with the services that you have chosen.
 
-### Manual installation
+### Nuxt project installation
 
```

Comment:
worth mentioning `@vue-storefront/nuxt` module as well

Reply:
Ok, I mentioned all 4 modules. However, we do not have Shopify nuxt module. Should I add it? If so, should it be part of this PR or another one? @filrak